### DISCUSSION TITLE
Question on importing photos / face rectangles

### DISCUSSION THREAD

**Discussion by Wonderwuzi2000** [RhetTbull/osxphotos#799] [Q&A]
Is it possible to import photos via osxphotos and include also face recognition/name of a person?
I mean something like the other way around as discussed at https://github.com/RhetTbull/osxphotos/discussions/632

**Comment by RhetTbull** [comment]
osxphotos can import using `osxphotos import` but it cannot also import the faces/persons associated with a photo. I had started a merge feature that would have enabled this in #356 but haven't worked on it due to time/difficulty/other projects.

**Reply by Wonderwuzi2000** [reply]
Thank you for your response.
The reason for my question was that I have a few pictures in my Apple Photos database where one person is recognized and named/displayed but a second person is listed as _UNKNOWN_ (not in "osxphotos inspect" but in "osxphotos persons") even though no face recognition rectangle shows up.

So I just wanted to get rid of these ghost entries via exporting, editing in digiKam an re-import those pictures.
I can send you an xmp file exported with "osxphotos export /Volumes/External/test --album "export" --sidecar XMP --sidecar-drop-ext" where you can find that no name is there for this second entry.

Maybe you have a solution.

PS:  osxphotos is a great pice of software - I would like to donate a few Euros/Dollars.
PPS: Sorry for my poor english but it's not my native language.

**Reply by RhetTbull** [reply]
No apology needed!  I'm glad you find osxphotos useful,. While I very much appreciate the offer of a donation, I don't currently accept donations for osxphotos -- this is a project I do to give back to the open source community.  Thank you for your consideration though!

I do not understand the issue you are having regarding faces.  `osxphotos persons` simply lists all the persons found in the entire database, not individual photos. `osxphotos inspect` does not show the unknown faces (these are filtered out by osxphotos).  Does a face rectangle show in Photos?

Is the problem that a face rectangle is showing up in DigiKam for the unknown person (via the XMP sidecar file)?  If so, a better solution would be to fix the XMP output so it does not include unknown faces.
